docs/device-preparation.md

```diff
@@ -27,6 +27,10 @@ Automatic configuration availability is limited on iOS, especially for a real de
 
 Please turn on _Web Inspector_ on iOS device via _Settings_ -> _Safari_ -> _Advanced_
 
+> **Note**
```

Comment:
Not sure if this should be a note. The suggestion is valid for all apps containing web views unless you are going to test Safari

Reply:
This is currently for ios 16.4 sdk built only, so currently set as note. hm, maybe can address SDK version as well..?